integration-tests/gradle/src/test/resources/kotlin-grpc-project/build.gradle

```diff
@@ -23,33 +23,16 @@ dependencies {
 group 'org.acme'
 version '1.0.0-SNAPSHOT'
 
-quarkus {
-    setOutputDirectory("$projectDir/build/classes/kotlin/main")
-}
-
-quarkusDev {
-    setSourceDir("$projectDir/src/main/kotlin")
-}
-
 allOpen {
     annotation("javax.ws.rs.Path")
     annotation("javax.enterprise.context.ApplicationScoped")
     annotation("io.quarkus.test.junit.QuarkusTest")
 }
 
-java {
-    sourceCompatibility = JavaVersion.VERSION_11
```

Comment:
Why was this removed?

Reply:
We used to not specify java version in sample project as Gradle test were ran against jdk 8. But this is not the case anymore. I don't know what if we should keep it or not..

Reply:
@glefloch 👆

Reply:
@ia3andy should I revert this change?

Reply:
@glefloch  I don't know 😅